website/docs/developer/development-guide/documentation/documentation-dev-env.md

```diff
@@ -19,13 +19,13 @@ The installation steps below goes into further details.
 
 1. Install the latest [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm) application to manage Node.js and npm versions.
    ```sh
-   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
    ```
 
-2. Install the proper Node.js and npm versions to your machine. Currently Node v14.17.6 should be used.
+2. Install the proper Node.js and npm versions to your machine. Currently Node v16.15.1 should be used.
```

Comment:
This comment conflicts with the `lts` version below.

Reply:
Reverted to use specific version in command.